WcaOnRails/db/migrate/20180623171213_add_registration_requirements_columns_to_competitions.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddRegistrationRequirementsColumnsToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :free_competition_enabled, :boolean
+    add_column :Competitions, :on_the_spot_registration_enabled, :boolean
+    add_column :Competitions, :on_the_spot_entry_fee, :boolean
+    add_column :Competitions, :on_the_spot_entry_fee_lowest_denomination, :integer, null: false, default: 0
+    add_column :Competitions, :refund_policy, :boolean
+    add_column :Competitions, :refund_policy_amount, :integer, null: false, default: 0
```

Comment:
I thought you were planning to do refund as a percentage? Did you change your mind about this?

Reply:
I was planning to do it as a percentage, but thought it would be clearer to users if we displayed this as a total amount. I'm open to change it to a percentage though.

Reply:
I think a fixed amount won't work well in a world where there are variable registration fees (such as per event fees).